invenio_app_ils/records/api.py

```diff
@@ -104,6 +104,16 @@ def relations(self):
         """Get record relations."""
         return self._relations
 
+    def clear(self):
+        """Clear IlsRecordWithRelations record."""
+        relations = self["relations"]
```

Comment:
you only need `relations_metadata`, `relation` field is normally a JSON resolver no?

See `class Series` `create` method:
* `relations` are added in the `create` method
* `tags` are actually not added, a but?

I think that resolver fields should be added in the `create` + `update` methods to ensure that they are there.
WDYT?

Reply:
@ntarocco `Relation` is indeed a jsonresolver. I am just thinking that if we move it in the update then you could not be able to override the field programmatically as it would always keep the first assigned value when it was created...I suppose this is fine. 

In series we have `tags`???